packages/datetime/test/dateRangeInputTests.tsx

```diff
@@ -177,6 +177,36 @@ describe("<DateRangeInput>", () => {
         });
     });
 
+    describe("allowSingleDayRange", () => {
+        it("allows start and end to be the same day when clicking", () => {
+            const { root, getDayElement } = wrap(<DateRangeInput
+                allowSingleDayRange={true}
+                defaultValue={[START_DATE, END_DATE]}
+            />);
+
+            // edit the end boundary
+            getEndInput(root).simulate("focus");
+
+            // deselect current end date
+            getDayElement(END_DAY).simulate("click");
+
```

Comment:
such whitespace. i mean whatever, but i like keeping blocks of interaction between asserts together.